### PR TITLE
Python 3: More use of six and update deps that won't install on 3.5

### PR DESCRIPTION
**Pull request by rowanseymour**
Contains #1040 which should be merged first

### CODE REVIEW

File: temba/channels/models.py [hunk 1]
```diff
@@ -624,7 +624,7 @@ def add_line_channel(cls, org, user, credentials, name):
 
     @classmethod
     def add_twitter_channel(cls, org, user, screen_name, handle_id, oauth_token, oauth_token_secret):
-        config = dict(handle_id=long(handle_id),
+        config = dict(handle_id=int(handle_id),
```
Comment (rowanseymour) [inline temba/channels/models.py]: In Python 2.7, `int` returns a `long` if it needs to. Int Python 3 there are only ints.

Comment (nicpottier) [inline temba/channels/models.py]: But won't this blow up now on 2.7 if it overflows an int?

Comment (rowanseymour) [inline temba/channels/models.py]: no `int(..)` just returns a `long`. It's weird like that.

```
type(int(12345678901234567890)) == long
```

Comment (nicpottier) [inline temba/channels/models.py]: Huh, weird and great!

**Pull request merged**